app/src/main/java/edu/grinnell/grinnell_publications_android/Fragments/ArticleFragment.java

```diff
@@ -0,0 +1,162 @@
+    private FloatingActionButton inviteButton;
+
+
+
+    ArticleFragment context = this;
+
+
+    /**
+     * Mandatory empty constructor for the fragment manager to instantiate the
+     * fragment (e.g. upon screen orientation changes).
+     */
+    public ArticleFragment() {
+    }
+
+    /**
+     * Constructor created by the newInstance and takes in the Item ID
+     */
+    public static ArticleFragment newInstance(){
+        ArticleFragment fragment = new ArticleFragment();
+        return fragment;
+    }
+    /*
+        public static DisplayFragment newInstance(Article article){
+        DisplayFragment fragment = new DisplayFragment();
+        return fragment;
+    }
+     */
+
+    /**
+     * OnCreate for the fragment
+     */
+    @Override
+    public void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+    }
+
+
+    /**
+     * OnCreateView that inflates and sets up the view
+     */
+    @Override
+    public View onCreateView(LayoutInflater inflater, ViewGroup container,
+                             Bundle savedInstanceState) {
+        final View DisplayFragmentView = inflater.inflate(R.layout.fragment_article,
+                container, false);
+
+        /* Setup view */
+        setViewItems(DisplayFragmentView);
+        toolbar.setNavigationIcon(getResources().getDrawable(R.drawable.ic_action_search)); //todo: Replace with backbutton
+        loadData();
+
+        /* logic setters */
+        setOnClickListeners();
+
+        return DisplayFragmentView;
+
+    }
+
+
+
+    /**
+     * SETUP METHOD:
+     * Takes the fragment view, sets each view item
+     */
+    private void setViewItems(View mView)
+    {
+
+        imageView = (ImageView) mView.findViewById(R.id.backdrop);
+        toolbar = (Toolbar) mView.findViewById(R.id.toolbar);
+        collapsingToolbar =
+                (CollapsingToolbarLayout) mView.findViewById(R.id.collapsing_toolbar);
+        inviteButton = (FloatingActionButton) mView.findViewById(R.id.invite);
+
+    }
+
+
+
+    /**
+     * SETUP METHOD:
+     * Sets the onclick listeners for each view item
+     */
+    public void setOnClickListeners() {
+
+        //onClick for back button
+        toolbar.setNavigationOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                getActivity().getSupportFragmentManager().popBackStack();
+            }
+        });
+
+        inviteButton.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                //onclick for the FAB
+            }
+        });
+
+    }
+
+    /**
+     * loads the data from article
+     */
+    private void loadData(){
+        //method for pulling data
+    }
+
+
+    /**
+     * Sets
+     */
+    private void setContent(String name, Date startDate, Date endDate, String imageURL, String description, String location){
```

Comment:
This method demands too much from the client calling this. As in, should I use this method, there are a  lot of things to remember. Please let's keep our method signatures to a decent length.